google/auth/transport/__init__.py

```diff
@@ -27,6 +27,10 @@
 import abc
 
 import six
+from six.moves import http_client
+
+DEFAULT_REFRESH_STATUS_CODES = (http_client.UNAUTHORIZED,)
+DEFAULT_MAX_REFRESH_ATTEMPTS = 2
```

Comment:
Do you want users to be able to tweak these? Do you want docstrings on these constants? (In theory every public constant should have a docstring.)